types/lerna__package/index.d.ts

```diff
@@ -52,20 +53,20 @@ export class Package {
     get optionalDependencies(): RawManifest['optionalDependencies'];
     get peerDependencies(): RawManifest['peerDependencies'];
     get<K extends (keyof RawManifest) | string>(key: K): RawManifest[K];
-    set<K extends (keyof RawManifest) | string>(key: K, val: RawManifest[K]): void;
+    set<K extends (keyof RawManifest) | string>(key: K, val: RawManifest[K]): this;
     /**
      * Provide shallow copy for munging elsewhere
      */
-    toJson(): object;
+    toJson(): RawManifest;
```

Comment:
based on the source file https://github.com/lerna/lerna/blob/main/core/package/index.js#L229
```suggestion
    toJSON(): RawManifest;
```